pkg/build/admission/admission.go

```diff
@@ -43,21 +43,57 @@ func NewBuildByStrategy(client client.Interface) admission.Interface {
 const (
 	buildsResource       = "builds"
 	buildConfigsResource = "buildconfigs"
+
+	cloneSubresource       = "clone"
```

Comment:
got rid of the subresource constants per @liggitt's comment below. So far it seems we're not using constants for anything else in master.go and boostrappolicy/policy.go, so I'll leave the 2 existing ones there since they're used at least twice in this file.